src/lib/components/Toggle/ToggleElement.js

```diff
@@ -0,0 +1,49 @@
+/*
+ * This file is part of React-SearchKit.
+ * Copyright (C) 2020 CERN.
+ *
+ * React-SearchKit is free software; you can redistribute it and/or modify it
+ * under the terms of the MIT License; see LICENSE file for more details.
+ */
+
+import React, { Component } from 'react';
+import { Checkbox } from 'semantic-ui-react';
+import Overridable from 'react-overridable';
+import _get from 'lodash/get';
+import { Card } from 'semantic-ui-react';
+
+
+class ToggleComponent extends Component{
+    constructor(props) {
+        super(props);
+        this.title = props.title;
+        this.label = props.label;
+        this.filterValue = props.filterValue;
+        this.updateQueryFilters = props.updateQueryFilters;
+      }
+
+      onToggleClicked = () => {
+        this.updateQueryFilters(this.filterValue);
+      };
+      
+      render() {
+        return(
+        <Overridable id={'ToggleComponent'}>
```

Comment:
Are you sure the overridable id is ok to be the same as the name of the component [here](https://github.com/inveniosoftware/react-searchkit/pull/145/files#diff-1afdea65744f0f08c5e62dbe48dbc8eaafd928c83968f554d4c0aca3e862bdcfR49) ? Also, we will need to pass down the props i.e `<Overridable id={'ToggleComponent'}> {...this.props}` so they can be available when you override the component I think....